rootfs/etc/confd/templates/builder

```diff
@@ -64,17 +64,30 @@ TMP_DIR=$(mktemp -d -p $BUILD_DIR)
 
 cd $REPO_DIR
 # use Procfile if provided, otherwise try default process types from ./release
+
+git archive --format=tar.gz ${GIT_SHA} > ${APP_NAME}.tar.gz
+
+tar -xzf ${APP_NAME}.tar.gz -C $TMP_DIR/
+
 USING_DOCKERFILE=true
-if [ -f Procfile ]; then
-    PROCFILE=$(cat Procfile | yaml2json-procfile)
+if [ -f $TMP_DIR/Procfile ]; then
+    PROCFILE=$(cat $TMP_DIR/Procfile | yaml2json-procfile)
     USING_DOCKERFILE=false
-    POD="slugbuilder"
 else
     PROCFILE="{}"
 fi
+# if [ -f Procfile ]; then
+#     PROCFILE=$(cat Procfile | yaml2json-procfile)
+#     USING_DOCKERFILE=false
+#     POD="slugbuilder"
+# else
+#     PROCFILE="{}"
+# fi
+echo "procfile" $PROCFILE
+echo "dockerfile" $USING_DOCKERFILE
 
 if [[ ! -f /var/run/secrets/object/store/access-key-id  ]]; then
-  if [ -f Dockerfile ]; then
+  if $USING_DOCKERFILE ; then
```

Comment:
regarding bash variables USING_DOCKERFILE is a global variable which is tru by default and being set as false if there is a procfile. Even though someone sets it as environment variable it might not affect the logic.